projects/update/client/updateProject.html

```diff
@@ -0,0 +1,8 @@
+<template name="updateProject">
+    <div class="page-header">
+      <h2>Update Project</h2>
+    </div>
+    
+    {{> quickForm collection="Projects" doc=getProject id="updateProjectForm" type="update"}}
```

Comment:
Shorten `getProject` to `project` for brevity and convention. I.e. we are generally using nouns for template helpers that return an object, as opposed to a verb phrase ('get project').